app/views/admin/shared/_logo_upload.html.erb

```diff
@@ -0,0 +1,12 @@
+ <div class="crayons-field mt-6 align-left" data-controller="logo-upload">
+  <%= label_tag :logo, class: "crayons-field__label" do %>
+    Logo
+    <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
```

Comment:
```suggestion
    <span class="crayons-field__required crayons-hover-tooltip" data-tooltip="This will set the logo for your Forem"></span>
```

refactor (blocking) - this tooltip doesn't appear, as for it to appear on hover, it needs to use the class `crayons-hover-tooltip`, as it's the 'hover only' variant being used here. I would question though if we need the tooltip or not - not all users are able to access it (only mouse users) and I think the intent of the field is pretty clear (and if it's not, maybe amending the label text would make more sense).

Note on the `aria-describedby` below 👇